fn_apility/README.md

```diff
@@ -0,0 +1,33 @@
+## About Apility.IO
+
+Apility.io can be defined as Threat Intelligence SaaS for developers and product companies that want to know in realtime if their existing or potential users have been classified as 'abusers' by one or more of these lists.
+
+Automatic extraction processes extracts all the information in realtime, keeping the most up to date data available, saving yourself the hassle of extract and update regularly all these lists and the data.
```

Comment:
I'm not sure sure what information this sentence is meant to convey. Maybe you can identify the different functions which this integration implements.

Reply:
I have put simple words. Plz check if missing anything.

Reply:
how about 

> Ability.io is an Internet accessible Threat Intelligence service for ....

Can you describe what 'these lists' mean? Is it ip addresses?